root-project.gradle

```diff
@@ -95,6 +95,27 @@ configure(subprojects) {
         exclude 'src/testUtil/java/com/google/firebase/firestore/testutil/ThrowingRunnable.java'
         exclude '**/package-info.java'
     }
+    apply plugin: 'jacoco'
+    jacoco {
+        toolVersion = '0.8.2'
```

Comment:
The latest version is 0.8.3. Can you see if this works with the latest?

Reply:
Done. Verified locally that 0.8.3 works.